src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
+    final private static String BOT_NAME = "Python";
+    final private static int HORIZONTAL_LINE_LENGTH = 80;
+
+    final private static String pythonAsciiArt =
```

Comment:
```suggestion
    final private static String PYTHONASCIIART =
```

Constants are recommended to be in ALL CAPS

Reply:
Changed! Thank you